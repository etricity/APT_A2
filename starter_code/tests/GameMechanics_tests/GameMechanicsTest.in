Starting Board:

  0    2    4    6    8
  ------------------------
A |    |    |    |    |    
B    |    |   P3   G4    |   
C |    |    |   P4    | 
D    |    |   Y4    |    |
E |    |    R4   |    |   
F    |    B4  R6    |    | 
G |    |    |    |    |   
  ------------------------

load
B5 P3
B7 G4
C6 P4
D5 Y4
E4 R4
F3 B4
F5 R6

play
G3 O6
G2 O4
E6 B6
E6 Y6
E0 B1
A8 O4
D7 P3
D7 P5