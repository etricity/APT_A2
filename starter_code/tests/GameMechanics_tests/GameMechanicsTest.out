Invalid Move
Not Qwirkle
0
Valid Move
Qwirkle!!
12
Invalid Move
Not Qwirkle
0
Valid Move
Not Qwirkle
4
Invalid Move
Not Qwirkle
0
Invalid Move
Not Qwirkle
0
Invalid Move
Not Qwirkle
0
Invalid Move
Not Qwirkle
0
